.github/workflows/ci.yml

```diff
@@ -304,6 +304,7 @@ jobs:
         config:
           - config-default
           - config-hdp3
+          - config-hive3
```

Comment:
exclude this for PR builds, like https://github.com/trinodb/trino/blob/ce2194ac4608c0b078a24da4805831df82baf7b2/.github/workflows/ci.yml#L335-L340

i added required label to the PR to still trigger the build